Updating onboarding and project information

choldgraf opened the pull request:
This is a refresh / addition to our onboarding and projects documentation. It takes a crack at several of the items here: https://github.com/2i2c-org/team-compass/issues/71

- Adds documentation about first places to look in the team compass (for those getting started)
- Adds an onboarding checklist as a github issue template and updates it
- Adds process about using the checklist + assigning an onboarding champion
- Adds pointers to the coordination page in the `get started` docs
- Updates our projects page with more recent information, as well as a list of hub infrastructure we currently run
- Updates some of the credentials etc that we need to create (though I suspect we may need to update this further)

Note: this adds a page with two tables representing our current hub deployment repositories, and the list of community hubs being served from `pilot-hubs`/: https://2i2c-team-compass--74.org.readthedocs.build/en/74/reference/hubs.html

If we like the list here, we should remove it from `pilot/` (though, I can see an argument for putting the latter list in `pilot-hubs/` rather than in team-compass)

choldgraf commented:
@damianavila asking for some 👀 from you since you are the most recent person to experience onboarding pain :-)

damianavila commented on .github/ISSUE_TEMPLATE/new-team-member.md:
```diff
@@ -0,0 +1,41 @@
+
+**Accounts**
+
+- [ ] Make email account
```
```suggestion
- [ ] Make email account
- [ ] Give access to the 2i2c Google Drive folder
```

choldgraf replied:
Ah I think that the reason that access had to be explicitly given was because the folder was under a non `@2i2c.org` account - I've moved that to the `@2i2c.org` account, which means that anybody w/ an `@2i2c.org` address should now be able to access it.

damianavila replied:
That's super!! Thanks!

damianavila commented on .github/ISSUE_TEMPLATE/new-team-member.md:
```diff
@@ -0,0 +1,41 @@
+
+**Permissions**
+
+- [ ] Add to appropriate Cloud projects
```
```suggestion
- [ ] Add to appropriate Google Cloud projects
```

damianavila commented on conf.py:
```diff
@@ -75,3 +75,50 @@
+    "https://raw.githubusercontent.com/2i2c-org/pilot-hubs/master/config/hubs/2i2c.cluster.yaml",
+    "https://raw.githubusercontent.com/2i2c-org/pilot-hubs/master/config/hubs/cloudbank.cluster.yaml",
+    "https://raw.githubusercontent.com/2i2c-org/pilot-hubs/master/config/hubs/paleohack.cluster.yaml",
+]
```
Don't you want to fetch all the `.cluster.yaml` here? Otherwise, this one will get outdated when we have a new cluster.

choldgraf replied:
Are there others that I'm missing?

damianavila replied:
> Ah yeah - maybe a better approach would be to use the github API to grab these files, rather than hard code the yaml files here.

Yep...

> Are there others that I'm missing?

I don't think so, just thinking into the future...

damianavila commented on get-started.md:
```diff
@@ -2,50 +2,73 @@
+## Accounts that should be created
 
-2i2c runs much of its cloud infrastructure at custom URLs created for the target institution. In order to ensure you have the ability to create new URLs, you should have access to the 2i2c NameCheap account. If not, ask for access in the Slack.
+- **A GMail address**. You can use this to access information in the 2i2c drive, as well as various online services that we use such as cloud engines. You can sign in to your email address at gmail.com (it behaves like a Google account).
```
Access to the 2i2c GDrive should be mentioned here, IMHO.

damianavila commented on get-started.md:
```diff
@@ -2,50 +2,73 @@
+## Accounts you should have access to
 
-### Cloud Accounts
+- **Google Drive**. Your 2i2c email address will get you access to a Google Drive that contains documents, brainstorms, and other assets that aren't fit for the 2i2c Team Compass. You can log into the Google Drive by going to `drive.google.com` and adding a new account using your `@2i2c.org` address and password.
```
Actually, you needed to give me access to the 2i2c folder, otherwise, the drive is empty for the onboardee.

choldgraf commented:
OK I believe that I addressed @damianavila's comments via docs changes + google drive changes. I:

- Changed the "current list of hubs" to use the github API instead of manually specifying the YML files (we'll need to update this again if/when the YML file structure changes)
- Updated the google drive language to make it clear that the folder needs to be added
- Added a note about Staff vs. collaborators

damianavila reviewed (approved):
I think this is ready now.

choldgraf commented:
wohoo! thanks for the review
